htmd/parameterization/cli.py

```diff
@@ -628,6 +625,12 @@ def main_parameterize(arguments=None):
     # Fit charges
     mol = _fit_charges(mol, args, qm_calculator)
 
+    # Recreate MM minimizer once charges have been fitted
+    mm_minimizer = None
+    if args.min_type == 'mm' or args.dihed_opt_type == 'mm':
```

Comment:
Why do you need the second condition? Isn't this enough:
```python
args.dihed_opt_type == 'mm':
```

Reply:
See above answer

Reply:
Changed